app/services/sp_return_url_resolver.rb

```diff
@@ -24,6 +24,14 @@ def homepage_url
     service_provider.return_to_sp_url
   end
 
+  def post_idv_followup_url
+    if service_provider.post_idv_followup_url.present?
+      UriService.add_params(service_provider.post_idv_followup_url, locale: I18n.locale)
```

Comment:
I'm a little wary of adding the locale parameter, other partner sites may not use the same structure? Would we eventually need to support configuring that as well?

Reply:
Yeah, I am wary of it as well. We may eventually need to support configuring it.